bundles/paikkatietoikkuna/admin/admin-layeranalytics/instance.js

```diff
@@ -219,6 +217,10 @@ Oskari.clazz.define('Oskari.framework.bundle.admin-layeranalytics.AdminLayerAnal
             return this.analyticsData;
         },
 
+        getSingleLayerData (id) {
+            return this.analyticsData.find((data) => data.id === id);
```

Comment:
This needs another XHR to work properly. See: https://github.com/nls-oskari/kartta.paikkatietoikkuna.fi/pull/156

Reply:
Check.